pkg/api/util/warnings.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2020 The cert-manager Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+// Warning values thrown by validating webhook
+// https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/
+const (
+	DeprecatedACMEEABKeyAlgorithmField = "ACME issuer spec field 'externalAccount.keyAlgorithm' is deprecated. The value of this field will be ignored."
```

Comment:
I think this could use a doc comment like:

```go
// DeprecatedACMEEABKeyAlgorithmField is raised when the deprecated keyAlgorithm field for an ACME external account binding (EAB) is given.
```

this would be flagged by `golint` if we were to add that tool to our CI pipeline (https://github.com/cert-manager/website/issues/538)

```text
exported const DeprecatedACMEEABKeyAlgorithmField should have comment (or a comment on this block) or be unexported
```

Reply:
Thanks- that's a really good suggestion I've added the comment.